diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/IdentifierNaming.kt

```diff
@@ -378,4 +390,14 @@ class IdentifierNaming(private val configRules: List<RulesConfig>) : Rule("ident
         val prevCatchKeyWord = parentValueParamList?.prevCodeSibling()?.elementType == CATCH_KEYWORD
         return node.text == "e" && node.findParentNodeWithSpecificType(CATCH) != null && prevCatchKeyWord
     }
+
+    enum class EnumStyles {
```

Comment:
May be rename it to "Style" and move it to StringCaseUtils?

Reply:
sure, but for enums I think we only need to support these two

Reply:
yes, that is fine, let's support only two for enums

Reply:
That's done